src/select-list-view.js

```diff
@@ -172,17 +191,28 @@ module.exports = class SelectListView {
   renderItems () {
     if (this.items.length > 0) {
       const className = ['list-group'].concat(this.props.itemsClassList || []).join(' ')
-      return $.ol(
-        {className, ref: 'items'},
-        ...this.items.map((item, index) => {
-          const selected = this.getSelectedItem() === item
 
-          return $(ListItemView, {
-            element: this.props.elementForItem(item, {selected, index}),
-            selected,
-            onclick: () => this.didClickItem(index)
+      if (this.visibilityObserver) {
+        etch.getScheduler().updateDocument(() => {
+          Array.from(this.refs.items.children).slice(this.props.initiallyVisibleItemCount).forEach(element => {
```

Comment:
I think you could add a `readAfterUpdate` method to the `SelectListView` and do this inside of that method instead of scheduling using `updateDocument`. etch will call your `readAfterUpdate` method at the appropriate time.

https://github.com/atom/etch#update-hooks

Reply:
I tried, but `readAfterUpdate` is called on `selectListView.update({items})` case, but not on `new SelectListView({items})`(I mean only on update), I'm not sure this method is appropriate for all use-case. Plus each ListItemView also use `getScheduler().updateDocument` approach. So I will keep this unchanged.